### DISCUSSION TITLE
How to remove ANSI coloring for Jenkins (Gauge + JS)

### DISCUSSION THREAD

**Discussion by jkchua** [getgauge/gauge#2310] [General]
Hello, 

I am currently switching the programming language from Java to JavaScript due to some technical limitations beyond my control. I completed the language change, but when I loaded them on Jenkins, it looked like gibberish. An example is below: 

`[0;36m# Customer Profile Web API
[0m[0;33m  ## Get Account Details	[0m[0;32m ✔[0m[0;32m ✔[0m
[0m
[0m[0;36m# Customer Profile Process API
[0m[0;33m  ## Get Account Details	[0m[0;32m ✔[0m[0;32m ✔[0m
[0m
[0m[0;36m# BSS/OSS Customer Account API
[0m[0;33m  ## Get Account Details	[0m[0;32m ✔[0m[0;32m ✔[0m[0;32m ✔[0m
[0m`

I noticed that it was the colouring that made this happen. However, when I used the command `simple-console`, it removes the letter P or F from the list. 

How do I retain the letter P/F and remove the colouring. :)

**Comment by sriv** [comment]
You can use the `--simple-console` flag, ex. `gauge run --simple-console specs`

**Reply by jkchua** [reply]
Hi Sriv, it's not showing the P and F. How can I determine which test cases have failed if it doesn't show anything? 

![image](https://user-images.githubusercontent.com/11019850/204197577-b1dcc93e-4075-42e1-a7b7-9081222586b7.png)

**Reply by haroon-sheikh** [reply]
If it's not throwing an exception than it has passed.
